src/Scenes/Scene.h

```diff
@@ -131,7 +131,9 @@ int Scene_getDescriptionWindow(lua_State* L);
 struct BackgroundMusic
 {
     std::string file;
-    float volume = 100.f;
+    float volume;
```

Comment:
The only comment I have about the code itself is that these variables have no initial value. Since C++11 you can now do it directly when you declare it, like:

```
float value = 1.0f;
bool loop = false;
bool enabled = true;
```

Or whatever you want the initial values to be.